addons/web/static/tests/helpers/utils.js

```diff
@@ -24,6 +24,10 @@ export function patchDate(year, month, day, hours, minutes, seconds) {
     var RealDate = window.Date;
     var actualDate = new RealDate();
     var fakeDate = new RealDate(year, month, day, hours, minutes, seconds);
+    const offset = luxon.Settings.defaultZone.offset();
+    if (offset) {
+        fakeDate.setMinutes(fakeDate.getMinutes() - offset - fakeDate.getTimezoneOffset());
+    }
```

Comment:
`luxon.Settings.defaultZone.offset();` must take an argument: https://moment.github.io/luxon/api-docs/index.html#zone

Maybe try something like that (I did not take too much time thinking about the truthiness of the following computation though):
```suggestion
    const offset = luxon.Settings.defaultZone.offset(fakeDate);
    fakeDate.setMinutes(fakeDate.getMinutes() - offset);
```

Reply:
OR, you could make the fakeDate with the proper timezone by calling:
```js
new Realdate(`{year}-{month}-{day}T{hour}:{minutes}:{seconds}{generateFixedTZStringForISO(offset)}`);
// write generateFixedTZStringForISO() to return strings like "+02:00" or "-04:00"
```

Reply:
It doesn't work, .offset() always returns the value set by patchTimeZone and not the difference. Probably, because patchTimeZone applies a fixed offset.